charts/consul/templates/client-config-configmap.yaml

```diff
@@ -24,7 +24,8 @@ data:
        in the UI. */}}
   config.json: |-
     {
-      "check_update_interval": "0s"
+      "check_update_interval": "0s",
+      "use_streaming_backend": false
```

Comment:
I put this here because it only causes issues when doing federation and consul connect